appservice/package.json

```diff
@@ -49,7 +49,7 @@
         "portfinder": "^1.0.25",
         "pretty-bytes": "^5.3.0",
         "simple-git": "1.132.0",
-        "vscode-azureextensionui": "^0.42.0",
+        "vscode-azureextensionui": "file:../ui/vscode-azureextensionui-0.42.0.tgz",
```

Comment:
Yeah I'm going to be doing this a lot now that `npm link` is having problems and I have to do `npm pack`/`npm install <path to tar>`. It's not like I can forget to update this part before merging because the build will always fail

Reply:
Yeah I was surprised that this actually works.  I usually just run `npm install <path_to_tar>` in the terminal

Reply:
Yeah but...doesn't that update your package.json to look like this?